packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -96,14 +97,27 @@ export interface ExpansionHub extends RevHub {
     getI2CReadStatus(i2cChannel: number): Promise<I2CReadStatus>;
 
     // Motor
+    /**
+     * Configure a specific motor
+     * @param motorChannel
+     * @param motorMode
+     * @param floatAtZero whether to coast when power is set to 0. If this is set to true, the motor will brake at 0 instead.
+     */
     setMotorChannelMode(
         motorChannel: number,
-        motorMode: number,
+        motorMode: MotorMode,
         floatAtZero: boolean,
     ): Promise<void>;
     getMotorChannelMode(
         motorChannel: number,
-    ): Promise<{ motorMode: number; floatAtZero: boolean }>;
+    ): Promise<{ motorMode: MotorMode; floatAtZero: boolean }>;
+
+    /**
+     * Enable a motor. The motor must be configured before enabling.
+     * @param motorChannel
+     * @param enable
+     */
+
     setMotorChannelEnable(motorChannel: number, enable: boolean): Promise<void>;
     getMotorChannelEnable(motorChannel: number): Promise<boolean>;
```

Comment:
This and the other motor-related getters need documentation

Reply:
Documented